src/Clock.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+
+export default class Clock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: new Date(),
+      isPickingHours: true,
+      isAm: false,
+      setHour: 12,
+      setMinute: '0',
+      setAmOrPm: 'AM',
```

Comment:
Starting the name with `set` is usually for setter functions. I recommend using `current` instead.